### PR TITLE
Update classifiers for pypi

### PR DESCRIPTION
**Pull request by janiversen**

### CODE REVIEW

**Comment by janiversen** [comment]
@dhoomakethu I believe this should do it. At least now all classifiers in the list you referenced.

**Comment by dhoomakethu** [comment]
Yes, I have the same modified file , the new error is
```
Content received from server:
<html>
 <head>
  <title>400 The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information.</title>
 </head>
 <body>
  <h1>400 The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information.</h1>
  The server could not comply with the request since it is either malformed or otherwise incorrect.<br/><br/>
The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information.


 </body>
</html>
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information.
```

**Comment by dhoomakethu** [comment]
Working on it

File: setup.cfg [hunk 1]
```diff
@@ -12,14 +12,19 @@ long_description = file: README.rst, CHANGELOG.rst, LICENSE
...
     Environment :: Console
-    Framework :: Twisted :: Tornado :: Asyncio
-    Intended Audience :: Developers :: Modbus testers
+    Framework :: Asyncio
+    Intended Audience :: Developers
     License :: OSI Approved :: BSD License
-    Operating System :: OS Independent :: Linux :: Windows :: Mac OS
-    Programming Language :: Python :: 3.8
-    Topic :: System :: Networking :: Utilities :: SCADA :: Modbus
+    Operating System :: OS Independent :: Linux
```
Comment (dhoomakethu) [inline setup.cfg:22]: There is no `Operating System :: OS Independent :: Linux` , we can use `Operating System :: POSIX :: Linux
`

Comment (dhoomakethu) [reply setup.cfg:22]: This is the working diff
```diff --git a/setup.cfg b/setup.cfg
index 6d4e6433..94c61680 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -9,17 +9,26 @@ platforms        = 'Linux', 'Mac OS X', 'Win'
 description      = "A fully featured modbus protocol stack in python"
 keywords         = 'modbus, asyncio, scada, client, server, simulator'
 long_description = file: README.rst, CHANGELOG.rst, LICENSE
-url              = 'https://github.com/riptideio/pymodbus/'
-classifiers      =
+url              = https://github.com/riptideio/pymodbus/
+classifiers      =
     Development Status :: 4 - Beta
-    Programming Language :: Python :: 3.8 :: 3.9 :: 3.10
     Environment :: Console
-    Framework :: Twisted :: Tornado :: Asyncio
-    Intended Audience :: Developers :: Modbus testers
+    Framework :: Twisted
+    Framework :: AsyncIO
+    Intended Audience :: Developers
     License :: OSI Approved :: BSD License
-    Operating System :: OS Independent :: Linux :: Windows :: Mac OS
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Operating System :: MacOS :: MacOS X
+    Operating System :: OS Independent
+    Operating System :: Microsoft
     Programming Language :: Python :: 3.8
-    Topic :: System :: Networking :: Utilities :: SCADA :: Modbus
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+    Topic :: System :: Networking
+    Topic :: Utilities
 project_urls     =
     Source Code  = https://github.com/riptideio/pymodbus
     Bug Reports  = https://github.com/riptideio/pymodbus/issues
(END)
```

Comment (janiversen) [reply setup.cfg:22]: OK for me. But should we really announce 3.11 and 3.12 ? I have not tested 3.11 and know there are problems with 3.12

Comment (janiversen) [reply setup.cfg:22]: Will you commit the changes directly ?

Comment (dhoomakethu) [reply setup.cfg:22]: Yes, I will do it once I figure out the issue with the `long_description`

Comment (janiversen) [reply setup.cfg:22]: Thanks, I have update README.rst to reflect that we now are at dev4.

Comment (dhoomakethu) [reply setup.cfg:22]: Cool , I have the release published to Pypi https://pypi.org/project/pymodbus/3.0.0.dev4/

Comment (janiversen) [reply setup.cfg:22]: Super, more testing for me 😃

**Pull request merged**